.github/workflows/update_dist.yml

```diff
@@ -1,19 +1,28 @@
+permissions:
+  id-token: write
 name: Update dist files
 on:
   push:
     branches:
       - main
 
 env:
-  GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
+  ESC_ACTION_OIDC_AUTH: true
+  ESC_ACTION_OIDC_ORGANIZATION: pulumi
+  ESC_ACTION_OIDC_REQUESTED_TOKEN_TYPE: urn:pulumi:token-type:access_token:organization
+  ESC_ACTION_ENVIRONMENT: imports/github-secrets
+  ESC_ACTION_EXPORT_ENVIRONMENT_VARIABLES: GITHUB_TOKEN=PULUMI_BOT_TOKEN
```

Comment:
Where can I find documentation for these environment variables?  Google doesn't seem to find anything, and I can't find anything on the `esc-actions` readme either.

Reply:
Commit history :/

I'll update the README